deblur/workflow.py

```diff
@@ -460,22 +547,33 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
                            "%s.derep" % basename(output_trim_fp))
     dereplicate_seqs(seqs_fp=output_trim_fp,
                      output_fp=output_derep_fp,
-                     min_size=min_size,
-                     uc_output=True)
+                     min_size=min_size)
     # Step 3: Remove artifacts
     output_artif_fp = remove_artifacts_seqs(seqs_fp=output_derep_fp,
                                             ref_fp=ref_fp,
                                             working_dir=working_dir,
                                             ref_db_fp=ref_db_fp,
                                             negate=negate,
                                             threads=threads)
+    if not output_artif_fp:
+        logger.debug('remove artifacts failed, aborting')
+        return
     # Step 4: Multiple sequence alignment
     output_msa_fp = join(working_dir,
                          "%s.msa" % basename(output_artif_fp))
-    with open(output_msa_fp, 'w') as f:
-        alignment = multiple_sequence_alignment(seqs_fp=output_artif_fp,
-                                                threads=threads)
-        f.write(alignment.to_fasta())
+    # with open(output_msa_fp, 'w') as f:
+    #     alignment = multiple_sequence_alignment(seqs_fp=output_artif_fp,
+    #                                             threads=threads)
+    #     if not alignment:
+    #         logger.debug('msa failed. aborting')
+    #         return False
+    #     f.write(alignment.to_fasta())
+    alignment = multiple_sequence_alignment(seqs_fp=output_artif_fp,
+                                            threads=threads)
+    if not alignment:
```

Comment:
Some samples can fail the MSA (e.g. when they have 1 read). In this case, we want to just fail on this sample and not stop the pipeline for the rest of the samples (just not include this sample in the final deblurred biom table). I think raising an error will stop the whole deblur script?